### DISCUSSION TITLE
Is it possible to add the container as a peer in wireguard app?

### DISCUSSION THREAD

**Discussion by arpitgupta** [qdm12/gluetun#737] [Q&A] [answered]
Curious if its possible to add the container that is running as a peer in the wireguard app, that way I can connect other devices on the lan to route throught the VPN as well. I see proxy options mentioned in the wiki. Having this ability would be nice as then one could use the official wireguard app which is available in various platforms.

**Comment by qdm12** [comment, marked as answer]
Maybe some day, but definitely a good suggestion! I'm currently very out of time and with a long backlog for this project. Please create a feature request issue so I can track it and you can get alerted on changes 😉  Thanks!

**Comment by arpitgupta** [comment]
Thanks logged #739
